test/tests/bootstrap/test_redfish10_os_bootstrap_parallel.py

```diff
@@ -0,0 +1,196 @@
+            log.error(" " + result['text'])
+            return False
+        if result['json']['TaskState'] == 'Running' or result['json']['TaskState'] == 'Pending':
+            if fit_common.VERBOSITY >= 2:
+                print "OS Install workflow state: {}".format(result['json']['TaskState'])
+            fit_common.time.sleep(30)
+        elif result['json']['TaskState'] == 'Completed':
+            if fit_common.VERBOSITY >= 2:
+                print "OS Install workflow state: {}".format(result['json']['TaskState'])
+            log.info_5(" " + result['text'])
+            return True
+        else:
+            log.error(" " + result['text'])
+            return False
+    log.error(" Workflow Timeout: " + result['text'])
+    return False
+
+# helper routine for selecting OS image path by matching proxy path
+def proxySelect(tag):
+    for entry in rackhdconfig['httpProxies']:
+        if tag in entry['localPath']:
+            return entry['localPath']
+    return ''
+
+# helper routine to return the task ID associated with the running bootstrap workflow
+def node_taskid(osname, version):
+    for entry in NODE_STATUS:
+        if NODE_STATUS[entry]['os'] == osname and NODE_STATUS[entry]['version'] == version:
+            return NODE_STATUS[entry]['id']
+    return ""
+
+# run all bootstraps if regression group is specified or no group is specified
+if "regression" in sys.argv or "-a" not in sys.argv:
+    oslist = [
+        {"os":"ESXi", "version":"5.5", "path":"/ESXi/5.5"},
+        {"os":"ESXi", "version":"6.0", "path":"/ESXi/6.0"},
+        {"os":"CentOS", "version":"6.5", "path":"/CentOS/6.5"},
+        {"os":"CentOS+KVM", "version":"6.5", "path":"/CentOS/6.5"},
+        {"os":"CentOS", "version":"7", "path":"/CentOS/7.0"},
+        {"os":"RHEL+KVM", "version":"7", "path":"/RHEL/7.0"},
+        {"os":"RHEL", "version":"7", "path":"/RHEL/7.0"},
+        {"os":"CentOS+KVM", "version":"7", "path":"/CentOS/7.0"}
+    ]
+    nodeindex = 0
+    for item in oslist:
+        # if OS proxy entry exists in RackHD config, run bootstrap against selected node
+        if proxySelect(item['path']) and nodeindex < len(NODECATALOG):
+            #delete active workflows for specified node
+            fit_common.cancel_active_workflows(NODECATALOG[nodeindex])
+            payload_data = {
+                            "osName": item['os'],
+                            "version": item['version'],
+                            "repo": rackhdhost + proxySelect(item['path']),
+                            "rootPassword": "1234567",
+                            "hostname": "rackhdnode",
+                            "dnsServers": [rackhdconfig['apiServerAddress']],
+                            "users": [{
+                                        "name": "rackhd",
+                                        "password": "R@ckHD1!",
+                                        "uid": 1010,
+                                    }]
+                           }
```

Comment:
If the os name contains `+KVM`, then the payload_data need an additional key-value pair `kvm: true`, otherwise, there is no difference between `xxx` and `xxx+KVM`.

Of course, if the node resource is limited, to skip those os name that ends with `+KVM` is also a choice.